modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -300,13 +319,43 @@ private PartialNode startPartialNode(
                 raftGroupEventsClientListener
         );
 
+        Map<String, Entry> metaStorageMockData = metaStorageMockDataByNode.computeIfAbsent(idx, k -> new HashMap<>());
+
+        var metaStorage = new RocksDbKeyValueStorage(name, dir.resolve("metastorage")) {
```

Comment:
I didn't get the point. Why we propagate RocksDbKeyValueStorage get/getAll to the metaStorageMockDataByNode instead of direct RocksDbKeyValueStorage calls?

Reply:
It's very tricky to imitate the node stop after partial handling of table creation, but before the assignments are written to meta storage in integration tests. This is, in fact, a sort of mock storage under the meta storage, making it behave as required on some calls of `metaStorage#getLocally()`.

Reply:
Still unclear to me. First of all

- Distributed ms get(), getAll() are propagated to local mocks. That seems a bit confusing. 
- Where do we put data to metaStorageMockData/metaStorageMockDataByNode? I've found only one place
```
        metaStorageMockData.put(
                new String(stablePartAssignmentsKey(new TablePartitionId(tableId, partId)).bytes(), StandardCharsets.UTF_8),
                new EntryImpl(null, null, 0, 0)
        );
```